src/app/blog/rss.xml/route.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/prefer-default-export */
+import Rss from 'rss';
+
+import { BLOG_BASE_PATH } from 'constants/blog';
+import { getAllWpPosts } from 'utils/api-posts';
+
+const SITE_URL = process.env.NEXT_PUBLIC_DEFAULT_SITE_URL;
+
+export async function GET() {
+  const allBlogPosts = await getAllWpPosts();
+
+  const feed = new Rss({
+    id: BLOG_BASE_PATH,
+    language: 'en',
+    title: `Blog — Neon Docs`,
+    description: 'The latest product updates from Neon',
+    feed_url: `${SITE_URL}${BLOG_BASE_PATH}rss.xml`,
```

Comment:
```suggestion
    feed_url: `${SITE_URL}${BLOG_BASE_PATH}/rss.xml`,
```
?

Reply:
I believe it is how the  BLOG_BASE_PATH looks like
 ```js
 const BLOG_BASE_PATH = '/blog/';
```